src/backend/gl/src/info.rs

```diff
@@ -295,13 +301,13 @@ impl Info {
 
 /// Load the information pertaining to the driver and the corresponding device
 /// capabilities.
-pub fn query_all(gl: &gl::Gl) -> (Info, Features, LegacyFeatures, Limits, PrivateCaps) {
+pub(crate) fn query_all(gl: &GlContainer) -> (Info, Features, LegacyFeatures, Limits, PrivateCaps) {
     use self::Requirement::*;
     let info = Info::get(gl);
 
     let mut limits = Limits {
-        max_texture_size: get_usize(gl, gl::MAX_TEXTURE_SIZE),
-        max_texel_elements: get_usize(gl, gl::MAX_TEXTURE_BUFFER_SIZE),
+        max_texture_size: get_usize(gl, gl::MAX_TEXTURE_SIZE).unwrap_or(0),
```

Comment:
should use the actual limits here

Reply:
What would be the actual limit for a failed `MAX_TEXTURE_BUFFER_SIZE`?

Reply:
buffer textures are 3.something, so 0 would be fine in this case as we don't support it at all for lower version. texture size should be supported for all and just in case for weird stuff we should use the minimum limit guarenteed by GL.

Reply:
Will do. :+1: